pkg/rancher-desktop/config/settingsImpl.ts

```diff
@@ -426,17 +449,15 @@ export const updateTable: Record<number, (settings: any) => void> = {
   8: (settings) => {
     // Rancher Desktop 1.10: move .extensions to .application.extensions.installed
     if (settings.extensions) {
-      settings.application ??= {};
-      settings.application.extensions ??= {};
-      settings.application.extensions.installed = settings.extensions;
+      _.set(settings, 'application.extensions.installed', settings.extensions);
       delete settings.extensions;
     }
```

Comment:
We might be able to use `processReplacements` here too?

Reply:
Yes